src/style.scss

```diff
@@ -0,0 +1,136 @@
+$main__color: #060b35;
+$secondary__color: #616070;
+$button__color: #00acdc;
+$white__color: #fff;
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body,
+div,
+h2,
+p,
+a {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: $white__color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main__color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary__color;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-amount {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    align-items: center;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary__color;
+  }
+
+  &__price-cost {
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main__color;
+  }
+
+  &__button {
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    background-color: $button__color;
+    border-radius: 5px;
+    color: $white__color;
+    border: 1px solid $button__color;
+    align-self: center;
+
+    &:hover {
+      background-color: $white__color;
+      color: $button__color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
In your html stars is a div element. div doesn't have any margins/paddings or list-style by default.